swagger.yaml

```diff
@@ -7,7 +7,7 @@ security:
   - BearerAuth: []
 paths:
 
-  /stories:
+  /v2/stories:
```

Comment:
Is there a specific reason to add `/v2` to the paths? Hasn't this been defined for the global base path at line 1475 of the file?